english/matrix/matrix.md

```diff
@@ -4,22 +4,22 @@ As the marshmallow reaches to grab the fork and rope
 The hand is moving slower than normal,
  as if it was in slo-mo.
 You merely walk over it and grab
- your rope and fork before the 
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.